web_src/less/_editor.less

```diff
@@ -12,9 +12,18 @@
   border-bottom: 0;
 }
 
+.repository.wiki .tab[data-tab="write"] .CodeMirror {
+  border-top: 0;
+}
+
 .editor-toolbar {
   opacity: 1 !important;
   border-color: var(--color-secondary);
+  position: sticky;
+  top: 0;
+  z-index: 10;
```

Comment:
I think `z-index` should be avoided because it's a unnecessary burden with the levels. Maybe [isolation](https://developer.mozilla.org/en-US/docs/Web/CSS/isolation) can be used instead to create a new stacking context instead.

Reply:
isolation: isolate only creates a new stack group, you'd still need to specify a relative z-index for it to work when using position: sticky; since the toolbar only should be on top of the editor and not much levels are specified on the page, any low value would suffice. Specifying a z-index of 1 could work, but call me old fashioned to use z-index of 10 :-).

isolation: isolate isn't required per se, and it doesn't hurt to specify it on it's wrapping container. But leaving it out completely breaks position: sticky.

Reply:
Ah I see, `position: sticky` already implicitely creates a stacking context so `isolation` would be redundant. If it works with `z-index: 1`, I guess that would be my preference.

Reply:
I'll push changes to update them to z-index: 1 for both files; They work, and as long as no-one sets the same value on it's siblings that's perfectly fine.